### ISSUE TITLE
Hent gjeldende plandokumenter med URL som retur?

### ISSUE THREAD

**Issue by sturtzel** [ks-no/fiks-plan-specification#105]
Antagelig en uplanlagt bieffekt av ryddingen i dokumentskjemaet:
Hvis man skal referere til planer er det en fordel å kunne benytte URL.
En ekstra tjeneste?
Et ekstra felt i tillegg til dokumentskjemaet for tjenesten for å returnere gjeldende dokumenter?

**Comment by jarleborsheim** [comment]
Har vi kanskje et større problem med tjenesten for "Hent dokumentfil"?
Den tar inn kun 1 nøkkel som er filnavn eller referanseDokumentfil som det står der nå. Det betyr altså at man må gi noe som er globalt unikt innenfor planregister. Det kan vel ikke fungere nå? 
Når man registrerer et dokument så gir man 2 atributter, filnavn og referanseDokumentfil. 
Filnavn er slik filen heter opprinnelig i systemet som sender meldingen, og referanseDokumentfil er filnavnet den har i meldingen (hvis det er flere filer med samme navn f.eks.). Men får ikke noe kvittering tilbake med noe unik id fra registrerende system. 
Men når man skal hente så må den som ønsker å hente meldingen ha en globalt unikt id å referere til eller så må man vel gi mer informasjon for å kunne hente korrekt fil?
```
{
    "$schema": "http://json-schema.org/draft-07/schema#",
    "$id": "no.ks.fiks.plan.v2.innsyn.dokumentfil.hent.schema.json",
    "description": "Henter en fil basert på referanseDokumentfil. Resultatet vil være en fil.",
    "title": "HentDokumentfil",
    "definitions": {},
    "type": "object",
    "properties": {
        "referanseDokumentfil": {
            "type": "string"
        }
    },
    "required": [
        "referanseDokumentfil"
    ]
}
```

**Comment by sturtzel** [comment]
no.ks.fiks.plan.v2.innsyn.gjeldendeplandokumenter.hent.resultat.schema.json:
Legge til feltet referanseDokumentfil?

Vi må også se på andre hent-tjenester.

**Comment by jarleborsheim** [comment]
Vi kan ikke legge til`referanseDokumentfil` som et felt i skjema no.ks.fiks.plan.v2.innsyn.gjeldendeplandokumenter.hent.resultat.schema.json da den returnerer en liste med objekter av typen dokument, se [her](https://github.com/ks-no/fiks-plan-specification/blob/main/Schema/V2/no.ks.fiks.plan.v2.innsyn.gjeldendeplandokumenter.hent.resultat.schema.json). Da må vi legge det til i dokument objektet. `referanseDokumentfil`er akkurat nå kun et filnavn, som ikke har noe unikt ved seg. Må vi ikke ta inn igjen `id` attributtet da, i `referanseDokumentfil`, som jeg kommenterte? Vi tar det opp på neste møte 👍

**Comment by jarleborsheim** [comment]
Vi legger til nytt felt for dokument objektet som heter id som må være unik i planregister. Vi lager et forslag. Og med en description som forklarer dette. Vi endrer også til id på dokumentfil hent tjenesten.
